### DISCUSSION TITLE
Community chat platform choice

### DISCUSSION THREAD

**Discussion by rdoursenaud** [mido/mido#549] [Polls]
Hi @mido community!

I'd like to set up a space for mido developers and users to gather in and exchange in real-time.

My current shortlist is:
- IRC on [LiberaChat](https://libera.chat/)
- [Matrix](https://matrix.org/) on [Gitter](https://gitter.im/)
- [Discord](https://discord.com/)

**Comment by bast** [comment]
No strong opinion but I have made excellent experience in other projects with https://zulip.com/why-zulip/ if we want one more alternative.

**Reply by rdoursenaud** [reply]
Thanks for the suggestion!
Looks like a good team chat.
Unfortunately I have excluded those (Slack, Google spaces…) from the list as I want something we can use publicly with the whole community and won't be limiting in the future.
I'd also like to not have to setup and maintain infrastructure even though I'm more than capable having done that professionally for many years.
Sorry for not outlining my rationale earlier.

**Comment by DanGylez** [comment]
Hello, rdoursenaud. I'm writing a Python program that uses MIDO to communicate with a Sound Generator and I have found myself struggling with an input issue wich causes it to freeze. Could you recomend me some place I could go looking for help from the community? Thanks!

**Reply by rdoursenaud** [reply]
Hey @DanGylez,
The chat hasn’t materialized yet.
Feel free to post your question here in the [Q&A section](https://github.com/orgs/mido/discussions/categories/q-a). I’ll do my best to help you out!

**Comment by nyktimeni** [comment]
**Discord** is OK, but **Matrix** or **Discourse** with a few choice plugins has my vote.
https://discourse.org
https://github.com/discourse/discourse

**Reply by rdoursenaud** [reply]
Thank you for your input!
I don’t feel the need for discourse yet as GitHub discussions seem fine for now for the asynchronous use case.
